src/main/java/com/dataguise/saas/exception/InternalServerException.java

```diff
@@ -0,0 +1,41 @@
+package com.dataguise.saas.exception;
+
+import org.springframework.http.HttpStatus;
+
+/**
+ * This exceptions are thrown when there is an error in processing from saas-ui backend to frontend.
+ */
+public class InternalServerException extends Exception {
+
+    String errorMessage;
+    HttpStatus status;
+
+    public InternalServerException() {}
+
+    public InternalServerException(String message, HttpStatus status) {
```

Comment:
**Reporter**: Checkstyle    
    **Rule**: com.puppycrawl.tools.checkstyle.checks.coding.HiddenFieldCheck
    **Severity**: ERROR
    **File**: /work/src/main/java/com/dataguise/saas/exception/InternalServerException.java L15
    
    &#39;status&#39; hides a field.
    
*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1832941045>*